types/database/schema/ChannelList.types.ts

```diff
@@ -42,6 +42,7 @@ export interface ChannelList {
   rawJson: any;
   user: UserSchema | null;
   members: ChannelListMemberSchema[] | null;
+  memberUsers: UserSchema[] | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `memberUsers: UserSchema[] | null;` is a good change as it provides a direct way to access the users in a channel. However, please ensure that this new property is populated correctly wherever a `ChannelList` object is created or updated.
```diff
+   memberUsers: UserSchema[] | null;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->